gammapy/irf/exposure.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from ..data import SpectralCube
+from astropy.coordinates import SkyCoord, Angle
+
+import logging
+
+__all__ = [
+    'exposure_cube'
+]
+
+log = logging.getLogger(__name__)
+
+
+def exposure_cube(pointing,
+                  livetime,
+                  aeff2D,
+                  ref_cube):
+    """Exposure cube
+
+        Parameters
```

Comment:
Detent this ... it should be at 4 spaces indentation just like the code inside the function.